README.md

```diff
@@ -23,6 +24,9 @@ The `TransformFunction` reads its configuration as `JSON` from the Function `use
 ```json
 {
   "steps": [
+    {
+      "type": "cast", "schema-type": "STRING"
```

Comment:
If you cat to string here, there's no point in dropping fields after

Reply:
good point, will restore the old order, I naively wanted to match the the above list.

Reply:
same applies to flatten and compute, maybe it not necessary to have a single example will possible steps. will remove for now, users can always read the doc on each step.